docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,114 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
+  sensible warning
+- [ ] Attempt to select an unsupported image
```

Comment:
Given that Etcher already filters the file-selection dialog by file-extension, it might be worth clarifying what "unsupported image" you should try selecting.
Two things I can think of off the top of my head are zip files containing more than one file inside them, and .tar.bz2 files. (perhaps there's others - should we try selecting all of them?)

Reply:
Done!